src/services/api.js

```diff
@@ -38,4 +38,21 @@ export const reportBug = (name, description) => {
         name,
         description
     })
+}
+
+export const register = (firstName, lastName, pesel, email, password, address) => {
+    return api.post(`/patient/registration`, {
+        firstName, 
+        lastName, 
+        pesel, 
+        email, 
+        password,
+        "address": {
+          "city": address.city,
+          "zipCode": address.zipCode,
```

Comment:
Po co te apostrofy? Jak przekazujesz zmienne o takiej samej nazwe to nie lepiej przekazac caly obiekt po prostu?